### PR TITLE
Add linting configs

### PR DESCRIPTION
**Pull request by siliconlad**

### CODE REVIEW

File: tool2schema/parameter_schema.py [hunk 1]
```diff
@@ -56,17 +56,14 @@ def _add_enum(self, schema: dict):
 
     def _add_description(self, schema: dict):
         """
-        Add the description of this parameter, extracted from the function docstring, to the given schema.
+        Add the description of this parameter, extracted from the function docstring, to the given
```
Comment (LorenzoCelli) [inline tool2schema/parameter_schema.py:59]: Was this supposed to take two lines?

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+indent_size = 4
+max_complexity = 10
+exclude =
+    # No need to traverse our git directory
+    .git,
+    # There's no value in checking cache directories
+    __pycache__,
+    # Contains logs which are not needed to be checked
+    log,
+    # Resource files that don't need checkin
```
Comment (LorenzoCelli) [inline .flake8:12]: Missing a g

**Review by LorenzoCelli — COMMENTED** [review]
Two very small things that can easily be ignored

File: tool2schema/parameter_schema.py (hunk 1 above)
Comment (siliconlad) [reply tool2schema/parameter_schema.py:59]: Yeah otherwise it exceeds the 100 character line limit.

File: .flake8 (hunk 2 above)
Comment (siliconlad) [reply .flake8]: Checkin 🤙 haha. Fixed now.

**Review by LorenzoCelli — APPROVED** [review]

**Pull request merged**